ironfish-cli/src/utils/transaction.ts

```diff
@@ -36,64 +35,68 @@ export async function watchTransaction(options: {
 
   const startTime = lastTime
 
-  if (last.content.transaction == null) {
-    logger.log(`Tried to watch transaction ${options.hash} but it's missing.`)
-    return
-  }
+  let prevStatus = last?.content.transaction?.status ?? TransactionStatus.NOT_FOUND
```

Comment:
I don't think we should add NOT_FOUND to TransactionStatus, since TransactionStatus is used for the status field of an existing transaction. In this case, we're just using it for display purposes.

Another way I think about it is that the `ironfish` package internally doesn't use the NOT_FOUND status, so I don't think we should change the enum just for the convenience of writing external (`ironfish-cli`) code.

I think you could type this as `type TransactionWatchStatus = TransactionStatus | 'not found'` and everything should just work.

Reply:
removed changes to `TransactionStatus`